extras/java8-tests/src/test/java/org/apache/spark/streaming/Java8APISuite.java

```diff
@@ -867,8 +865,8 @@ public void testTrackStateByAPI() {
     JavaPairRDD<String, Boolean> initialRDD = null;
```

Comment:
nit: the method name `testTrackStateByAPI` should be renamed to `testMapWithStateAPI`